packages/use-dataloader/src/DataLoaderProvider.tsx

```diff
@@ -59,8 +58,8 @@ export const DataLoaderContext = createContext<IDataLoaderContext>(undefined)
 const DataLoaderProvider = ({
   children,
   cacheKeyPrefix,
-  onError,
-  maxConcurrentRequests,
+  onError = undefined,
```

Comment:
The type should be `onError?: OnErrorFn` and then we can remove this `= undefined`

Reply:
yes i already made the change ;)